concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2331,6 +2332,38 @@ jobs:
                   cf blue-green-deploy paas-metrics
                   cf delete -f paas-metrics-old
 
+        - task: deploy-paas-uaa-assets
```

Comment:
In a previous retro we decided that we should try to remove some of the tasks from `cf-deploy` because it had grown too large, but we couldn't refactor all of them in one go so we'd do it incrementally. Do you think this is a good candidate to start with, or not because UAA is dependent on it?

Reply:
Would that be in the direction of having one task that deploys all the apps? It would make sense to start here.

Reply:
One task to deploy all the apps has no separation and it'd be scary to change later. We could deploy `rubbernecker`, `paas-dashboard` and *maybe* `paas-metrics` using the CI environment, but I'm not sure what the tradeoffs are.

I feel this extra task starts to make that work necessary, but this task does belong in core `cf-deploy` because it's directly tied to UAA. The refactoring might be best as a maintenance story?

Reply:
I agree that rubbernecker should move to CI, but the other app you mention form part of the platform itself, so IMO should be deployed by the pipeline.

It's worth noting that when https://www.pivotaltracker.com/story/show/152539815 has been played, `paas-dashboard` won't exist any more,

Reply:
I think it's fine to leave as-is for now because this app isn't a good example of something that's independent from `cf-deploy` - UAA won't function correctly without the app - whereas the other tasks like service brokers and monitoring don't have such hard/circular dependencies. Thought it was worth a discussion though.